extlinks/links/management/commands/linkevents_collect.py

```diff
@@ -45,7 +45,22 @@ def handle(self, *args, **options):
         else:
             url = base_stream_url
 
-        for event in EventSource(url):
+        # Eventsource should fail if it can't read data after ~15 minutes.
+        for event in EventSource(
+            url,
+            # The retry argument sets the delay between retries in milliseconds.
+            # We're setting this to 5 minutes.
+            # There's no way to set the max_retries value with this library,
+            # but since it depends upon requests, which in turn uses urllib3
+            # by default, we get a default max_retries value of 3.
```

Comment:
This may be a bit nit-picky, but I would rather have this comment be a whole string comment like so

```suggestion
         """The retry argument sets the delay between retries in milliseconds.
             We're setting this to 5 minutes.
            There's no way to set the max_retries value with this library,
            but since it depends upon requests, which in turn uses urllib3
            by default, we get a default max_retries value of 3."""
```

Reply:
Works for me! I was just sticking with the style already in the file, but this is better.